platform/WORKSPACE

```diff
@@ -163,7 +163,7 @@ go_repository(
 
 go_repository(
     name = "io_k8s_kube_openapi",
-    commit = "0cf8f7e6ed1d2e3d47d02e3b6e559369af24d803",
+    commit = "c59034cc13d587f5ef4e85ca0ade0c1866ae8e1d",
```

Comment:
out of curiosity, why this commit?

Reply:
I grabbed them from the Godeps in the apimachinery, client-go, and api repositories.